org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java

```diff
@@ -185,12 +186,29 @@ private void runAnalysisAndUpdateMarkers(@Nullable Server server, Map<ISonarLint
     Map<ISonarLintIssuable, List<Issue>> issuesPerResource = new LinkedHashMap<>();
     filesToAnalyze.keySet().forEach(slFile -> issuesPerResource.put(slFile, new ArrayList<>()));
 
+    long start = System.currentTimeMillis();
     AnalysisResults result = run(server, config, issuesPerResource, monitor);
     if (!monitor.isCanceled() && result != null) {
       updateMarkers(server, docPerFiles, issuesPerResource, result, triggerType, monitor);
+      updateTelemetry(inputFiles, start);
     }
   }
 
+  private void updateTelemetry(List<ClientInputFile> inputFiles, long start) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "updateTelemetry" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.eclipse%3Asonarlint-eclipse-parent&pullRequest=142&issues=AWKWRRav-RUO4oGVXYul&open=AWKWRRav-RUO4oGVXYul)

[ISSUE_KEY]: AWKWRRav-RUO4oGVXYul